README.md

```diff
@@ -11,6 +13,46 @@ npm install --save analytics-client
 
 ## Usage
 
+Tracking a page view. The following reports an event named `[EFP] Page View` with default properties.
```

Comment:
Where does the `EFP` bit come from? I can't see it below at all so I assume it's using some external context?

Reply:
[#L29](https://github.com/balena-io-modules/analytics-client/blame/575e4cf7091a190f8e72eab5ec70238de31d5d2d/README.md#L29) sets the prefix `createWebTracker(client, 'EFP')`.

Reply:
Indeed right now only web tracker supports event name prefix. It's best to allow any tracker to configure such prefix. I created an issue about this: https://github.com/balena-io-modules/analytics-client/issues/36